texastribune/salesforce-stripe
# Blast

What this PR does
+ Changes the Blast subscription form from a SelectField to RadioField
+ Updates the relevant scripts to account for the template change in the form
+ Fixes a javascript bug that didn't catch new value when the form subscription field changed

How to test:
+ Check that the styles look good on various sizes/screen dimensions
+ Check that when you switch values, the pay fees recalculates correctly and the Stripe processor picks up the correct value
+ Make sure you can still submit
